From 4be052e1200ef9a574e499bb00b31a1d861bc219 Mon Sep 17 00:00:00 2001
From: Rob Herring <rob.herring@calxeda.com>
Date: Fri, 17 Aug 2012 09:41:04 -0500
Subject: [PATCH 329/951] ARM: highbank: disable unused sdhci and gpio in dts

These peripherals are not enabled in current systems, so turn them off.

Signed-off-by: Rob Herring <rob.herring@calxeda.com>
Acked-by: Arnd Bergmann <arnd@arndb.de>
---
 arch/arm/boot/dts/highbank.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/highbank.dts b/arch/arm/boot/dts/highbank.dts
index 0c6fc34..28d5872 100644
--- a/arch/arm/boot/dts/highbank.dts
+++ b/arch/arm/boot/dts/highbank.dts
@@ -132,6 +132,7 @@
 			reg = <0xffe0e000 0x1000>;
 			interrupts = <0 90 4>;
 			clocks = <&eclk>;
+			status = "disabled";
 		};
 
 		memory-controller@fff00000 {
@@ -156,6 +157,7 @@
 			interrupts = <0 14 4>;
 			clocks = <&pclk>;
 			clock-names = "apb_pclk";
+			status = "disabled";
 		};
 
 		gpiof: gpio@fff31000 {
@@ -166,6 +168,7 @@
 			interrupts = <0 15 4>;
 			clocks = <&pclk>;
 			clock-names = "apb_pclk";
+			status = "disabled";
 		};
 
 		gpiog: gpio@fff32000 {
@@ -176,6 +179,7 @@
 			interrupts = <0 16 4>;
 			clocks = <&pclk>;
 			clock-names = "apb_pclk";
+			status = "disabled";
 		};
 
 		gpioh: gpio@fff33000 {
@@ -186,6 +190,7 @@
 			interrupts = <0 17 4>;
 			clocks = <&pclk>;
 			clock-names = "apb_pclk";
+			status = "disabled";
 		};
 
 		timer {
-- 
1.8.0

